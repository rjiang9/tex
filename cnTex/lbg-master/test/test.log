This is XeTeX, Version 3.1415926-2.2-0.9995.2 (TeX Live 2009/Debian) (format=xelatex 2012.10.23)  27 OCT 2012 16:56
entering extended mode
 %&-line parsing enabled.
**test.tex
(./test.tex
LaTeX2e <2009/09/24>
Babel <v3.8l> and hyphenation patterns for english, usenglishmax, dumylang, noh
yphenation, pinyin, loaded.
(/usr/share/texmf-texlive/tex/latex/base/book.cls
Document Class: book 2007/10/19 v1.4h Standard LaTeX document class
(/usr/share/texmf-texlive/tex/latex/base/bk10.clo
File: bk10.clo 2007/10/19 v1.4h Standard LaTeX file (size option)
)
\c@part=\count80
\c@chapter=\count81
\c@section=\count82
\c@subsection=\count83
\c@subsubsection=\count84
\c@paragraph=\count85
\c@subparagraph=\count86
\c@figure=\count87
\c@table=\count88
\abovecaptionskip=\skip41
\belowcaptionskip=\skip42
\bibindent=\dimen102
)
(/usr/share/texmf-texlive/tex/latex/graphics/graphicx.sty
Package: graphicx 1999/02/16 v1.0f Enhanced LaTeX Graphics (DPC,SPQR)

(/usr/share/texmf-texlive/tex/latex/graphics/keyval.sty
Package: keyval 1999/03/16 v1.13 key=value parser (DPC)
\KV@toks@=\toks14
)
(/usr/share/texmf-texlive/tex/latex/graphics/graphics.sty
Package: graphics 2009/02/05 v1.0o Standard LaTeX Graphics (DPC,SPQR)

(/usr/share/texmf-texlive/tex/latex/graphics/trig.sty
Package: trig 1999/03/16 v1.09 sin cos tan (DPC)
)
(/etc/texmf/tex/latex/config/graphics.cfg
File: graphics.cfg 2009/08/28 v1.8 graphics configuration of TeX Live
)
Package graphics Info: Driver file: xetex.def on input line 91.

(/usr/share/texmf-texlive/tex/xelatex/xetex-def/xetex.def
File: xetex.def 2009/11/22 v0.94 LaTeX color/graphics driver for XeTeX (RRM/JK)

))
\Gin@req@height=\dimen103
\Gin@req@width=\dimen104
)
(/usr/share/texmf-texlive/tex/xelatex/fontspec/fontspec.sty
Package: fontspec 2008/08/09 v1.18 Advanced font selection for XeLaTeX

(/usr/share/texmf-texlive/tex/generic/ifxetex/ifxetex.sty
Package: ifxetex 2009/01/23 v0.5 Provides ifxetex conditional
)
\c@zf@newff=\count89
\c@zf@index=\count90
\c@zf@script=\count91
\c@zf@language=\count92

(/usr/share/texmf-texlive/tex/latex/tools/calc.sty
Package: calc 2007/08/22 v4.3 Infix arithmetic (KKT,FJ)
\calc@Acount=\count93
\calc@Bcount=\count94
\calc@Adimen=\dimen105
\calc@Bdimen=\dimen106
\calc@Askip=\skip43
\calc@Bskip=\skip44
LaTeX Info: Redefining \setlength on input line 76.
LaTeX Info: Redefining \addtolength on input line 77.
\calc@Ccount=\count95
\calc@Cskip=\skip45
)
(/usr/share/texmf-texlive/tex/latex/xkeyval/xkeyval.sty
Package: xkeyval 2008/08/13 v2.6a package option processing (HA)

(/usr/share/texmf-texlive/tex/generic/xkeyval/xkeyval.tex
\XKV@toks=\toks15
\XKV@tempa@toks=\toks16
\XKV@depth=\count96
File: xkeyval.tex 2008/08/13 v2.6a key=value parser (HA)
))
(/usr/share/texmf-texlive/tex/latex/base/fontenc.sty
Package: fontenc 2005/09/27 v1.99g Standard LaTeX package

(/usr/share/texmf-texlive/tex/xelatex/euenc/eu1enc.def
File: eu1enc.def 2008/03/08 v0.1d Experimental unicode font encoding
)
LaTeX Font Info:    Try loading font information for EU1+lmr on input line 100.


(/usr/share/texmf-texlive/tex/xelatex/euenc/eu1lmr.fd
File: eu1lmr.fd 2007/01/14 v1.3 Font defs for Latin Modern
))
LaTeX Info: Redefining \itshape on input line 1163.
LaTeX Info: Redefining \slshape on input line 1166.
LaTeX Info: Redefining \scshape on input line 1169.
LaTeX Info: Redefining \upshape on input line 1172.

fontspec.cfg loaded.
(/usr/share/texmf-texlive/tex/xelatex/fontspec/fontspec.cfg))
\c@zf@famc@Adobe Fangsong Std R=\count97
Package fontspec Info: Defining font family for 'Adobe Fangsong Std R' with opt
ions [] on input line 4.
Package fontspec Info: Could not resolve font Adobe Fangsong Std R/B (it might 
not exist) on input line 4.
Package fontspec Info: Could not resolve font Adobe Fangsong Std R/I (it might 
not exist) on input line 4.
Package fontspec Info: Could not resolve font Adobe Fangsong Std R/BI (it might
 not exist) on input line 4.


LaTeX Warning: Unused global option(s):
    [UTF-8,hyperref].

(./test.aux)
\openout1 = `test.aux'.

LaTeX Font Info:    Checking defaults for OML/cmm/m/it on input line 9.
LaTeX Font Info:    ... okay on input line 9.
LaTeX Font Info:    Checking defaults for T1/cmr/m/n on input line 9.
LaTeX Font Info:    ... okay on input line 9.
LaTeX Font Info:    Checking defaults for OT1/cmr/m/n on input line 9.
LaTeX Font Info:    ... okay on input line 9.
LaTeX Font Info:    Checking defaults for OMS/cmsy/m/n on input line 9.
LaTeX Font Info:    ... okay on input line 9.
LaTeX Font Info:    Checking defaults for OMX/cmex/m/n on input line 9.
LaTeX Font Info:    ... okay on input line 9.
LaTeX Font Info:    Checking defaults for U/cmr/m/n on input line 9.
LaTeX Font Info:    ... okay on input line 9.
LaTeX Font Info:    Checking defaults for EU1/lmr/m/n on input line 9.
LaTeX Font Info:    ... okay on input line 9.
Package fontspec Info: Adjusting the maths setup (use [no-math] to avoid this).
 on input line 9.
\symlegacymaths=\mathgroup4
LaTeX Font Info:    Overwriting symbol font `legacymaths' in version `bold'
(Font)                  OT1/cmr/m/n --> OT1/cmr/bx/n on input line 9.
LaTeX Font Info:    Redeclaring math accent \acute on input line 9.
LaTeX Font Info:    Redeclaring math accent \grave on input line 9.
LaTeX Font Info:    Redeclaring math accent \ddot on input line 9.
LaTeX Font Info:    Redeclaring math accent \tilde on input line 9.
LaTeX Font Info:    Redeclaring math accent \bar on input line 9.
LaTeX Font Info:    Redeclaring math accent \breve on input line 9.
LaTeX Font Info:    Redeclaring math accent \check on input line 9.
LaTeX Font Info:    Redeclaring math accent \hat on input line 9.
LaTeX Font Info:    Redeclaring math accent \dot on input line 9.
LaTeX Font Info:    Redeclaring math accent \mathring on input line 9.
LaTeX Font Info:    Redeclaring math symbol \colon on input line 9.
LaTeX Font Info:    Redeclaring math symbol \Gamma on input line 9.
LaTeX Font Info:    Redeclaring math symbol \Delta on input line 9.
LaTeX Font Info:    Redeclaring math symbol \Theta on input line 9.
LaTeX Font Info:    Redeclaring math symbol \Lambda on input line 9.
LaTeX Font Info:    Redeclaring math symbol \Xi on input line 9.
LaTeX Font Info:    Redeclaring math symbol \Pi on input line 9.
LaTeX Font Info:    Redeclaring math symbol \Sigma on input line 9.
LaTeX Font Info:    Redeclaring math symbol \Upsilon on input line 9.
LaTeX Font Info:    Redeclaring math symbol \Phi on input line 9.
LaTeX Font Info:    Redeclaring math symbol \Psi on input line 9.
LaTeX Font Info:    Redeclaring math symbol \Omega on input line 9.
LaTeX Font Info:    Redeclaring math symbol \mathdollar on input line 9.
LaTeX Font Info:    Redeclaring symbol font `operators' on input line 9.
LaTeX Font Info:    Encoding `OT1' has changed to `EU1' for symbol font
(Font)              `operators' in the math version `normal' on input line 9.
LaTeX Font Info:    Overwriting symbol font `operators' in version `normal'
(Font)                  OT1/cmr/m/n --> EU1/AdobeFangsongStdR(0)/m/n on input l
ine 9.
LaTeX Font Info:    Encoding `OT1' has changed to `EU1' for symbol font
(Font)              `operators' in the math version `bold' on input line 9.
LaTeX Font Info:    Overwriting symbol font `operators' in version `bold'
(Font)                  OT1/cmr/bx/n --> EU1/AdobeFangsongStdR(0)/m/n on input 
line 9.
LaTeX Font Info:    Overwriting symbol font `operators' in version `normal'
(Font)                  EU1/AdobeFangsongStdR(0)/m/n --> EU1/AdobeFangsongStdR(
0)/m/n on input line 9.
LaTeX Font Info:    Overwriting math alphabet `\mathrm' in version `normal'
(Font)                  EU1/AdobeFangsongStdR(0)/m/n --> EU1/AdobeFangsongStdR(
0)/m/n on input line 9.
LaTeX Font Info:    Overwriting math alphabet `\mathit' in version `normal'
(Font)                  OT1/cmr/m/it --> EU1/AdobeFangsongStdR(0)/m/it on input
 line 9.
LaTeX Font Info:    Overwriting math alphabet `\mathbf' in version `normal'
(Font)                  OT1/cmr/bx/n --> EU1/AdobeFangsongStdR(0)/bx/n on input
 line 9.
LaTeX Font Info:    Overwriting math alphabet `\mathsf' in version `normal'
(Font)                  OT1/cmss/m/n --> EU1/lmss/m/n on input line 9.
LaTeX Font Info:    Overwriting math alphabet `\mathtt' in version `normal'
(Font)                  OT1/cmtt/m/n --> EU1/lmtt/m/n on input line 9.
LaTeX Font Info:    Overwriting symbol font `operators' in version `bold'
(Font)                  EU1/AdobeFangsongStdR(0)/m/n --> EU1/AdobeFangsongStdR(
0)/bx/n on input line 9.
LaTeX Font Info:    Overwriting math alphabet `\mathrm' in version `bold'
(Font)                  EU1/AdobeFangsongStdR(0)/m/n --> EU1/AdobeFangsongStdR(
0)/bx/n on input line 9.
LaTeX Font Info:    Overwriting math alphabet `\mathit' in version `bold'
(Font)                  OT1/cmr/bx/it --> EU1/AdobeFangsongStdR(0)/bx/it on inp
ut line 9.
LaTeX Font Info:    Overwriting math alphabet `\mathsf' in version `bold'
(Font)                  OT1/cmss/bx/n --> EU1/lmss/bx/n on input line 9.
LaTeX Font Info:    Overwriting math alphabet `\mathtt' in version `bold'
(Font)                  OT1/cmtt/m/n --> EU1/lmtt/bx/n on input line 9.
LaTeX Font Info:    Font shape `EU1/AdobeFangsongStdR(0)/bx/n' undefined
(Font)              using `EU1/AdobeFangsongStdR(0)/m/n' instead on input line 
16.
File: ./icon.png Graphic file (type QTm)
 <use  "./icon.png" > [1


] [2

]
LaTeX Font Info:    External font `cmex10' loaded for size
(Font)              <12> on input line 48.
LaTeX Font Info:    External font `cmex10' loaded for size
(Font)              <8> on input line 48.
LaTeX Font Info:    External font `cmex10' loaded for size
(Font)              <6> on input line 48.
 [1] [2

] (./test.toc
LaTeX Font Info:    External font `cmex10' loaded for size
(Font)              <7> on input line 3.
LaTeX Font Info:    External font `cmex10' loaded for size
(Font)              <5> on input line 3.
)
\tf@toc=\write3
\openout3 = `test.toc'.

 [3] [4

] [5]
[6]
Chapter 1.

Overfull \hbox (16.38663pt too wide) in paragraph at lines 73--78
 \EU1/AdobeFangsongStdR(0)/bx/n/10 HHelloHelloHelloHelloHelloHelloHelloH[] ello
HelloHelloHelloHelloHello HelloHel-
 []


Overfull \hbox (9.08238pt too wide) in paragraph at lines 73--78
\EU1/AdobeFangsongStdR(0)/bx/n/10 loHelloHelloHelloHelloHelloHell oHelloHelloHe
lloHelloHelloH elloHelloHelloHel-
 []


Overfull \hbox (18.72662pt too wide) in paragraph at lines 81--90
 \EU1/AdobeFangsongStdR(0)/bx/n/10 HHelloHelloHelloHelloHelloHelloHelloH[] lloH
elloHello HHelloHelloHelloHelloHel-
 []


Overfull \hbox (9.08238pt too wide) in paragraph at lines 81--90
\EU1/AdobeFangsongStdR(0)/bx/n/10 loHelloHelloH elloHelloHelloHelloHelloHello H
elloHelloHelloHelloHelloHelloHel-
 []


Overfull \hbox (10.94905pt too wide) in paragraph at lines 81--90
\EU1/AdobeFangsongStdR(0)/bx/n/10 loHell oHelloHelloHelloHelloHelloH elloHelloH
elloHelloHelloHelloHelloHell oHel-
 []


Overfull \hbox (8.6133pt too wide) in paragraph at lines 92--97
[]\EU1/AdobeFangsongStdR(0)/bx/n/10 elloHelloHelloHelloHelloHello HelloHelloHel
loHelloHelloHelloHelloHell oHel-
 []


Overfull \hbox (9.08238pt too wide) in paragraph at lines 92--97
\EU1/AdobeFangsongStdR(0)/bx/n/10 loHelloHelloHelloHelloH elloHelloHelloHelloHe
lloHelloHelloHell oHelloHelloHel-
 []


Overfull \hbox (8.6133pt too wide) in paragraph at lines 99--104
[]\EU1/AdobeFangsongStdR(0)/bx/n/10 elloHelloHelloHelloHelloHello HelloHelloHel
loHelloHelloHelloHelloHell oHel-
 []


Overfull \hbox (9.08238pt too wide) in paragraph at lines 99--104
\EU1/AdobeFangsongStdR(0)/bx/n/10 loHelloHelloHelloHelloH elloHelloHelloHelloHe
lloHelloHelloHell oHelloHelloHel-
 []


Overfull \hbox (8.6133pt too wide) in paragraph at lines 106--111
[]\EU1/AdobeFangsongStdR(0)/bx/n/10 elloHelloHelloHelloHelloHello HelloHelloHel
loHelloHelloHelloHelloHell oHel-
 []


Overfull \hbox (9.08238pt too wide) in paragraph at lines 106--111
\EU1/AdobeFangsongStdR(0)/bx/n/10 loHelloHelloHelloHelloH elloHelloHelloHelloHe
lloHelloHelloHell oHelloHelloHel-
 []


Overfull \hbox (8.6133pt too wide) in paragraph at lines 113--118
[]\EU1/AdobeFangsongStdR(0)/bx/n/10 elloHelloHelloHelloHelloHello HelloHelloHel
loHelloHelloHelloHelloHell oHel-
 []


Overfull \hbox (9.08238pt too wide) in paragraph at lines 113--118
\EU1/AdobeFangsongStdR(0)/bx/n/10 loHelloHelloHelloHelloH elloHelloHelloHelloHe
lloHelloHelloHell oHelloHelloHel-
 []

[7

] [8

] [9] [10]
Chapter 2.
[11

] [12

]
Appendix A.
[13] [14

]
Appendix B.
[15] (./test.aux)
LaTeX Font Info:    Some font shapes were not available, defaults substituted.
 ) 
Here is how much of TeX's memory you used:
 2600 strings out of 495713
 49810 string characters out of 1188882
 127915 words of memory out of 3000000
 5736 multiletter control sequences out of 15000+50000
 6480 words of font info for 36 fonts, out of 3000000 for 9000
 28 hyphenation exceptions out of 8191
 27i,8n,32p,464b,265s stack positions out of 5000i,500n,10000p,200000b,50000s

Output written on test.pdf (17 pages).
