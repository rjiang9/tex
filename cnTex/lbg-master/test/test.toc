\contentsline {part}{I\hspace {1em}第一部分}{V}
\contentsline {chapter}{\numberline {1}测试}{VII}
\contentsline {section}{\numberline {1.1}hello, world}{VII}
\contentsline {subsection}{\numberline {1.1.1}hell.....}{VII}
\contentsline {paragraph}{This is the para title}{VII}
\contentsline {paragraph}{HHelloHelloHelloHelloHelloHelloHelloH}{VII}
\contentsline {paragraph}{HHelloHelloHelloHelloHelloHelloHelloH}{VII}
\contentsline {part}{II\hspace {1em}第2部分}{IX}
\contentsline {chapter}{\numberline {2}2}{XI}
\contentsline {chapter}{\numberline {A}hello A}{XIII}
\contentsline {chapter}{\numberline {B}B}{XV}
