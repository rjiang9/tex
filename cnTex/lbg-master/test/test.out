\BOOKMARK [-1][-]{part.1}{I 第一部分}{}
\BOOKMARK [0][-]{chapter.1}{测试}{part.1}
\BOOKMARK [1][-]{section.1.1}{hello, world}{chapter.1}
\BOOKMARK [2][-]{subsection.1.1.1}{hell.....}{section.1.1}
\BOOKMARK [-1][-]{part.2}{II 第2部分}{}
\BOOKMARK [0][-]{chapter.2}{2}{part.2}
\BOOKMARK [0][-]{appendix.A}{hello A}{part.2}
\BOOKMARK [0][-]{appendix.B}{B}{part.2}
